webrtc/lib/webrtc/variant_selector.ex

```diff
@@ -366,8 +395,10 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.VariantSelector do
 
   defp best_active_variant(selector) do
     selector.active_variants
+    |> Enum.filter(fn variant ->
+      fits_in_allocation?(selector, variant) || selector.muted_variant == variant
```

Comment:
> || selector.muted_variant == variant

This can never happen as muted variant is removed from active variants, am I right?

Reply:
This happens when the previously paused variant becomes active. Normally we would discard it, as it likely doesn't fit the allocation (allocation is 0) but we allow it anyway because it was paused.